test/SignerBondsUniswapV2.test.js

```diff
@@ -0,0 +1,166 @@
+    )
+
+    signerBondsUniswapV2 = await SignerBondsUniswapV2.deploy(
+      uniswapV2RouterStub.address,
+      coveragePool.address
+    )
+    await signerBondsUniswapV2.deployed()
+
+    const fakeAddress = "0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    const RiskManagerV1Stub = await ethers.getContractFactory(
+      "RiskManagerV1Stub"
+    )
+    riskManagerV1 = await RiskManagerV1Stub.deploy(
+      fakeAddress,
+      fakeAddress,
+      fakeAddress,
+      signerBondsUniswapV2.address,
+      fakeAddress,
+      86400,
+      75
+    )
+    await riskManagerV1.deployed()
+
+    await governance.sendTransaction({
+      to: riskManagerV1.address,
+      value: ethers.utils.parseEther("20"),
+    })
+  })
+
+  describe("approveSwapper", () => {
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(thirdParty)
+            .approveSwapper(swapper.address)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when caller is the governance", () => {
+      let tx
+      beforeEach(async () => {
+        tx = await signerBondsUniswapV2
+          .connect(governance)
+          .approveSwapper(swapper.address)
+      })
+
+      it("should approve the swapper", async () => {
+        expect(await signerBondsUniswapV2.approvedSwappers(swapper.address)).to
+          .be.true
+      })
+
+      it("should emit RiskManagerUnapproved event", async () => {
+        await expect(tx)
+          .to.emit(signerBondsUniswapV2, "SignerBondsSwapperApproved")
+          .withArgs(swapper.address)
+      })
+    })
+  })
+
+  describe("unapproveSwapper", () => {
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(thirdParty)
+            .unapproveSwapper(swapper.address)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when swapper is not approved", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(governance)
+            .unapproveSwapper(swapper.address)
+        ).to.be.revertedWith("Signer swapper is not approved")
+      })
+    })
+
+    context("when caller is governance and swapper approved", () => {
+      let tx
+      beforeEach(async () => {
+        await signerBondsUniswapV2
+          .connect(governance)
+          .approveSwapper(swapper.address)
+
+        tx = await signerBondsUniswapV2
+          .connect(governance)
+          .unapproveSwapper(swapper.address)
+      })
+
+      it("should unapprove the swapper", async () => {
+        expect(await signerBondsUniswapV2.approvedSwappers(swapper.address)).to
+          .be.false
+      })
+
+      it("should emit RiskManagerUnapproved event", async () => {
```

Comment:
s/RiskManagerUnapproved/SignerBondsSwapperUnapproved

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/148/commits/8b06db86517b95422ace7846320b14a1cbaee844